cosmwasm/deployment/terra2/tools/deploy.js

```diff
@@ -251,7 +240,19 @@ for (const [contract, registrations] of Object.entries(
         memo: "",
       })
       .then((tx) => terra.tx.broadcast(tx))
-      .then((rs) => console.log(rs));
+      .then((rs) => console.log(rs))
+      .catch((error) => {
```

Comment:
we're catching the error and logging, but not rethrowing - this means the exit code will be success even when deployment fails, right? shouldn't we exit with an error in that case?

Reply:
sure, makes sense to exit with an error. Since the token bridge and the contract registrations are effectively needed to run on devnet, there's no reason to consider it the deployment a success when this call fails.